slackhealthbot/domain/models/activity.py

```diff
@@ -12,16 +24,16 @@ class ActivityZone(StrEnum):
 @dataclasses.dataclass
 class ActivityZoneMinutes:
     zone: ActivityZone
-    minutes: int
+    minutes: Metric
 
 
 @dataclasses.dataclass
 class ActivityData:
     log_id: int
     type_id: int
     name: str
-    total_minutes: int
-    calories: int
+    total_minutes: Metric
+    calories: Metric
```

Comment:
Revert the changes in this commit.

Instead of making these attributes of type `Metric`:
* Adapt `ActivityHistory` to include the info for all-time top scores, and recent top scores
* Adapt `usecase_process_new_activity` to get the top scores from the repository
* Make `usecase_process_new_activity` merge the top scores with the `ActivityData` from the service, into an `ActivityHistory`
* Adapt `usecase_post_activity` to read the top scores info from `ActivityHistory` to format the message sent to slack.